tasks/broad/Alignment.wdl

```diff
@@ -17,6 +17,76 @@ version 1.0
 
 import "../../structs/dna_seq/DNASeqStructs.wdl"
 
+task Bazam {
+  input {
+    File input_bam
+    String bwa_commandline
+    String output_bam_basename
```

Comment:
What would you think of reading and writing CRAMs directly?

Reply:
Definitely can do that, yes!